lib/cylc/cfgspec/suite.py

```diff
@@ -200,32 +200,24 @@ def _coerce_final_cycletime(value, keys, args):
             '__MANY__': vdr(vtype='string'),
         },
         'event hooks': {
-            'startup handler': vdr(
-                vtype='string_list',
-                default=GLOBAL_CFG.get([
-                    'cylc', 'event hooks', 'startup handler'])),
-            'timeout handler': vdr(
-                vtype='string_list',
-                default=GLOBAL_CFG.get([
-                    'cylc', 'event hooks', 'timeout handler'])),
-            'shutdown handler': vdr(
-                vtype='string_list',
-                default=GLOBAL_CFG.get([
-                    'cylc', 'event hooks', 'shutdown handler'])),
-            'timeout': vdr(
-                vtype='interval_minutes',
-                default=GLOBAL_CFG.get(['cylc', 'event hooks', 'timeout'])),
-            'reset timer': vdr(vtype='boolean', default=True),
+            'handlers': vdr(vtype='string_list'),
+            'handler events': vdr(vtype='string_list'),
+            'startup handler': vdr(vtype='string_list'),
+            'timeout handler': vdr(vtype='string_list'),
```

Comment:
The logic has been moved to `Scheduler._get_events_conf` in `cylc.scheduler` for run time flexiblility.